src/block-kit/composition/Mrkdwn.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx JSXSlack.h */
+import { JSXSlack } from '../../jsx'
+import { ObjectOutput } from '../../utils'
+import html from '../../html'
+import { BlockComponentProps } from '../Blocks'
+
+interface MrkdwnProps extends BlockComponentProps {
```

Comment:
Don't extend `BlockComponentProps`. It includes `blockId` prop (and `id` alias prop) for [layout blocks](https://api.slack.com/reference/block-kit/blocks), and `<Mrkdwn>` composition object is not a part of that.